PowerEditor/src/WinControls/Preference/preferenceDlg.cpp

```diff
@@ -3189,7 +3560,7 @@ void BackupSubDlg::updateBackupGUI()
 		isEnableGlobableCheck = true;
 		isEnableLocalCheck = BST_CHECKED == ::SendDlgItemMessage(_hSelf, IDC_BACKUPDIR_CHECK, BM_GETCHECK, 0, 0);
 	}
-	::EnableWindow(::GetDlgItem(_hSelf, IDC_BACKUPDIR_USERCUSTOMDIR_GRPSTATIC), isEnableGlobableCheck);
+	//::EnableWindow(::GetDlgItem(_hSelf, IDC_BACKUPDIR_USERCUSTOMDIR_GRPSTATIC), isEnableGlobableCheck);

```

Comment:
@donho
again the same issue with groupbox, in light mode it does nothing, in dark mode reset color to black

if you need some changes, could it be done in separate PR, I won't available for a while, and I don't want block this PR